Increase bucket creation timeout for test_recovery_from_volume_deletion_cli_tool

RP- https://url.corp.redhat.com/8ad772a

Test failed on 4.18 with

```
E           ocs_ci.ocs.exceptions.UnhealthyBucket: rgw-oc-bucket-ba8ff0154ba648a4a8d9612639 did not reach a healthy state within 180 seconds.
E           OBC YAML:
E           {
E             "apiVersion": "objectbucket.io/v1alpha1",
E             "kind": "ObjectBucketClaim",
E             "metadata": {
E               "creationTimestamp": "2025-02-24T23:07:50Z",
E               "finalizers": [
E                 "objectbucket.io/finalizer"
E               ],
E               "generation": 1,
E               "labels": {
E                 "bucket-provisioner": "openshift-storage.ceph.rook.io-bucket"
E               },
E               "name": "rgw-oc-bucket-ba8ff0154ba648a4a8d9612639",
E               "namespace": "openshift-storage",
E               "resourceVersion": "195376",
E               "uid": "68dae49d-4eb6-4b59-a213-e5ea4dbabd4e"
E             },
E             "spec": {
E               "bucketName": "rgw-oc-bucket-ba8ff0154ba648a4a8d9612639",
E               "storageClassName": "ocs-storagecluster-ceph-rgw"
E             },
E             "status": {
E               "phase": "Pending"
E             }
E           }
E
E           OBC description:
E           Name:         rgw-oc-bucket-ba8ff0154ba648a4a8d9612639
E           Namespace:    openshift-storage
E           Labels:       bucket-provisioner=openshift-storage.ceph.rook.io-bucket
E           Annotations:  
E           API Version:  objectbucket.io/v1alpha1
E           Kind:         ObjectBucketClaim
E           Metadata:
E             Creation Timestamp:  2025-02-24T23:07:50Z
E             Finalizers:
E               objectbucket.io/finalizer
E             Generation:        1
E             Resource Version:  195376
E             UID:               68dae49d-4eb6-4b59-a213-e5ea4dbabd4e
E           Spec:
E             Bucket Name:         rgw-oc-bucket-ba8ff0154ba648a4a8d9612639
E             Storage Class Name:  ocs-storagecluster-ceph-rgw
E           Status:
E             Phase:  Pending
E           Events:   



ocs_ci/ocs/resources/objectbucket.py:302: UnhealthyBucket
```

@sagihirshfeld suggested to increase the timeout for bucket creation.

--- a/ocs_ci/ocs/resources/objectbucket.py
+++ b/ocs_ci/ocs/resources/objectbucket.py
@@ -267,7 +267,7 @@ class ObjectBucket(ABC):
             logger.error(f"{self.name} was not deleted within {timeout} seconds.")
             assert False, f"{self.name} was not deleted within {timeout} seconds."
 
-    def verify_health(self, timeout=60, interval=5, **kwargs):
+    def verify_health(self, timeout=800, interval=5, **kwargs):
         """
         Health verification function that tries to verify
         the a bucket's health by using its appropriate internal_verify_health

